src/main/java/emu/grasscutter/game/avatar/Avatar.java

```diff
@@ -432,11 +427,13 @@ public Set<Integer> getTalentIdList() {
     }
 
     public int getCoreProudSkillLevel() {
-        return coreProudSkillLevel;
```

Comment:
Should be safe to remove the `Avatar.coreProudSkillLevel` property now, since it's no longer used anywhere.